routes/category.js

```diff
@@ -0,0 +1,30 @@
+const express = require("express");
+const middleWare = require("../middleWare/getCategory");
+const categoryController = require("../controllers/categoryController");
+const router = express.Router();
+
+//get all categories
+router.get("/", categoryController.categoryIndex);
+
+//post a category to db
+router.post("/post", categoryController.addCategoryToDb);
```

Comment:
No need to mention /post, /delete etc in your URI, REST verbs like the post, put, get, patch, and delete themselves are self-explanatory about the operation they are going to perform.
Instead, you just need to mention on what resource(Resource should be a noun and use its plural form here) these operations will be performed, for ex:
router.post("/categories") // To create a category
router.get("/categories/:id") // To get a category for given id
router.get("/categories") // To get all categories
router.delete("/categories/:id") // To delete a given category
router.put("/categories/:id") // To update category for given id